app/ante/evm/09_gas_consume_test.go

```diff
@@ -86,29 +90,28 @@ func (suite *EvmAnteTestSuite) TestUpdateCumulativeGasWanted() {
 func (suite *EvmAnteTestSuite) TestConsumeGasAndEmitEvent() {
 	keyring := testkeyring.New(1)
 	unitNetwork := network.NewUnitTestNetwork(
+		network.WithChainID(suite.chainID),
 		network.WithPreFundedAccounts(keyring.GetAllAccAddrs()...),
 	)
 	grpcHandler := grpc.NewIntegrationHandler(unitNetwork)
 
 	testCases := []struct {
 		name          string
 		expectedError string
-		fees          sdktypes.Coins
+		feesAmt       math.Int
 		getSender     func() sdktypes.AccAddress
 	}{
+		// {
+		// 	name: "success: fees are zero and event emitted",
+		// 	fees: sdktypes.Coins{},
+		// 	getSender: func() sdktypes.AccAddress {
+		// 		// Return prefunded sender
+		// 		return keyring.GetKey(0).AccAddr
+		// 	},
+		// },
```

Comment:
why was this commented out?

Reply:
Good catch @MalteHerrmann! Just reintroduced it 🙏